src/views/neutralview/neutral-view-sanskrit.js

```diff
@@ -18,19 +18,24 @@ export class NeutralViewSanskrit extends LitElement {
             <p><b>The Sanskrit data will be uploaded soon.</b></p>
             <p>
               The textual corpus used in BuddhaNexus was obtained from the
-              Göttingen Register of Electronic Texts in Indian Languages
-              (GRETIL) for Sanskrit texts. Due to the huge amount of material,
-              some texts from the GRETIL database have been omitted (cumulative
-              pāda indexes and duplicate texts from the same source). Moreover,
-              there has been no attempt by BuddhaNexus to improve the quality of
-              the texts (e.g. removing typos, introducing identical conventions,
-              and the like). Some minor changes have, nonetheless, been made for
-              the sake of standardization.
+              Göttingen Register of Electronic Texts in Indian Languages (<a
+                href="http://gretil.sub.uni-goettingen.de/gretil.html"
+                >GRETIL</a
+              >) for Sanskrit texts. Due to the huge amount of material, some
+              texts from the GRETIL database have been omitted (cumulative pāda
+              indexes and duplicate texts from the same source). Moreover, there
+              has been no attempt by BuddhaNexus to improve the quality of the
+              texts (e.g. removing typos, introducing identical conventions, and
+              the like). Some minor changes have, nonetheless, been made for the
+              sake of standardization. In order to make the matching process
+              feasible, some markup information of the original GRETIL files has
+              been neglected.
             </p>
             <p>
-              For the calculation of the Sanskrit matches, a stemming algorithm
-              has been used. This stemming algorithm is accessible as a
-              standalone application <a href="/tools/sanskrit">here</a>.
+              For the calculation of the Sanskrit matches and for the global
+              search function, a stemming algorithm has been used. This stemming
+              algorithm is accessible as a standalone application
+              <a href="/sanskrit-tools">here</a>.
```

Comment:
Sorry, but I hate it when people make "here" the linktext! May I change it to linking 'standalone application' and leave out the word 'here'.

Reply:
Also needs styling in line with the rest of the site.